### PR TITLE
Backtrace in libafl::Error

### PR DESCRIPTION
**Pull request by andreafioraldi**
#571 

errors_backtrace is the feature to use it, disabled by default

### CODE REVIEW

File: libafl/src/mutators/mopt_mutator.rs [hunk 1]
```diff
@@ -340,7 +340,7 @@ impl MOpt {
             || (res + 1 < operator_num
                 && select_prob > self.probability_now[self.swarm_now][res + 1])
         {
-            return Err(Error::MOpt("Error in select_algorithm".to_string()));
+            return Err(Error::mopt("Error in select_algorithm".to_string()));
```
Comment (domenukk) [inline libafl/src/mutators/mopt_mutator.rs:343]: Unrelated, but does an extra mopt error really make sense?

Comment (tokatoka) [reply libafl/src/mutators/mopt_mutator.rs:343]: It's not helpful.
This error message is copy-pasted from the thesis-author's repo.

Comment (domenukk) [reply libafl/src/mutators/mopt_mutator.rs]: Removed it

**Comment by andreafioraldi** [comment]
cargo-tarpaulin, dunno what it is, but it fails

**Pull request merged**